Tests/OpenAPIRuntimeTests/URICoder/Serialization/Test_URISerializer.swift

```diff
@@ -140,6 +148,9 @@ final class Test_URISerializer: Test_Runtime {
             try testVariant(.simpleUnexplode, testCase.variants.simpleUnexplode)
             try testVariant(.formDataExplode, testCase.variants.formDataExplode)
             try testVariant(.formDataUnexplode, testCase.variants.formDataUnexplode)
+            if let deepObjectExplode = testCase.variants.deepObjectExplode {
```

Comment:
Why is this if statement here instead of running the variant unconditionally like the previous few?

Reply:
Yeah, I don't like it either. My problem here is the scenario that we are testing arrays serialisation. 
I managed to add an `expectedError` on the makeCase in order to handle errors too.